tmt/convert.py

```diff
@@ -713,7 +722,7 @@ def read_nitrate_case(testcase, makefile_data=None):
     except (KeyError, TypeError):
         pass
     for bug in testcase.bugs:
-        add_bug(bug.bug, data)
+        add_bug(bug.bug, data, bug.system)
```

Comment:
@hegerj does this mean tcms testcase will contain magic variables `1` or `2` for system? I would rather see some strings maybe?

Reply:
This is how `nitrate` currently implements bugzilla/jira links, from the API you `1` for bugzilla bugs and `2` for jira issues.